code/modules/mob/living/carbon/human/species/station/monkey.dm

```diff
@@ -95,13 +97,14 @@
 	flesh_color = "#34AF10"
 	base_color = "#066000"
 	reagent_tag = IS_UNATHI
+	holder_type = /obj/item/weapon/holder/monkey/stok
 
 /datum/species/monkey/bug
 	name = "V'krexi"
 	short_name = "kre"
 	name_plural = "V'krexi"
 	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/bug
-
```

Comment:
@LordFowl Let me know if you make, have-made, or find suitable held icons for vkrexi, ill happily add them